### ISSUE TITLE
[Issue]: Jellyfin does't start 

### ISSUE THREAD

**Issue by rushirufu** [jellyfin/jellyfin#9358]
### Please describe your bug

[root@localhost downloads]# systemctl status jellyfin
× jellyfin.service - Jellyfin Media Server
     Loaded: loaded (/usr/lib/systemd/system/jellyfin.service; disabled; preset: disabled)
    Drop-In: /etc/systemd/system/jellyfin.service.d
             └─override.conf
     Active: failed (Result: exit-code) since Sun 2023-02-19 15:45:36 EST; 6s ago
   Duration: 47ms
    Process: 1679 ExecStart=/usr/bin/jellyfin $JELLYFIN_WEB_OPT $JELLYFIN_RESTART_OPT $JELLYFIN_FFMPEG_OPT $JELLYFIN_SERVICE_OPT $JE>   Main PID: 1679 (code=exited, status=137)
        CPU: 28ms

Feb 19 15:45:36 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 5.
Feb 19 15:45:36 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:45:36 localhost.localdomain systemd[1]: jellyfin.service: Start request repeated too quickly.
Feb 19 15:45:36 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:45:36 localhost.localdomain systemd[1]: Failed to start jellyfin.service - Jellyfin Media Server.

[root@localhost downloads]#  journalctl -u jellyfin
Feb 19 15:06:02 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:02 localhost.localdomain jellyfin[2942]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:02 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:02 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 1.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2944]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 2.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2946]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 3.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2948]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 4.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
lines 1-24...skipping...
Feb 19 15:06:02 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:02 localhost.localdomain jellyfin[2942]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:02 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:02 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 1.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2944]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 2.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2946]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 3.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2948]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 4.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:04 localhost.localdomain jellyfin[2950]: Failed to create CoreCLR, HRESULT: 0x80004005
lines 1-26...skipping...
Feb 19 15:06:02 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:02 localhost.localdomain jellyfin[2942]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:02 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:02 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 1. 
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2944]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 2.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2946]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 3.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain jellyfin[2948]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:03 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 4.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:03 localhost.localdomain systemd[1]: Started jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:04 localhost.localdomain jellyfin[2950]: Failed to create CoreCLR, HRESULT: 0x80004005
Feb 19 15:06:04 localhost.localdomain systemd[1]: jellyfin.service: Main process exited, code=exited, status=137/n/a
Feb 19 15:06:04 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:04 localhost.localdomain systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 5.
Feb 19 15:06:04 localhost.localdomain systemd[1]: Stopped jellyfin.service - Jellyfin Media Server.
Feb 19 15:06:04 localhost.localdomain systemd[1]: jellyfin.service: Start request repeated too quickly.
Feb 19 15:06:04 localhost.localdomain systemd[1]: jellyfin.service: Failed with result 'exit-code'.
Feb 19 15:06:04 localhost.localdomain systemd[1]: Failed to start jellyfin.service - Jellyfin Media Server.
### Jellyfin Version

10.8.0

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by certainlysylvia** [comment]
Had the same issue - if I turn Selinux off - issue goes away. Of course, havign Selinux off is not an option.

**Comment by Shadowghost** [comment]
Please fill in the issue template.

**Comment by certainlysylvia** [comment]
go to home or wherever you keep self generated selinux modules and do this
`
ausearch -c 'jellyfin' --raw | audit2allow -M my-jellyfin
semodule -X 300 -i my-jellyfin.pp
`

my-jellyfin.pp can be named whatever you want.

cheers - is working so far. Will update tomorrow :D

**Comment by certainlysylvia** [comment]
Ok faster than tomorrow.

I got another issue re nis so do this if you get that alert ( PS: Cockpit is very useful - use it!)
'
setsebool -P nis_enabled 1
'
typed in bash or whatever you use.

I will wait on further issues until tomorrow and report all at once.

**Comment by certainlysylvia** [comment]
Okay - still had some issue and I could tell this was going to become a long long list of mods - so went back to an old solution ( I thought this issue had been fixed, since my install worked for a day)

https://www.reddit.com/r/jellyfin/comments/xz4x0u/issues_starting_under_systemd/

that ABOLUTELY solves it - nice catch and kudos to the poster
